github-for-jira.sd.yml

```diff
@@ -147,6 +147,22 @@ resources:
         # Name of the RDS resource from above
         resource: rds
 
+  - type: lambda
+    name: auto-deployment-github-app
+    attributes:
+      runtime: nodejs14.x
+      prefetchMicrosEnvVars: true
+      artifact: "github-for-jira/auto-deployment-github-app-${BITBUCKET_BUILD_NUMBER}.zip"
+      handler: auto-deployment.handler
+      private: true
+      concurrentExecutions: 2
+      memory: 128
```

Comment:
This is the default value, so we can just remove this line https://hello.atlassian.net/wiki/spaces/MICROS/pages/505840672/Lambda+Resource+Type#memory

Reply:
`memory` has been removed